### ISSUE TITLE
[Visual Requirements - Make Code - Extensions] Elements must have sufficient color contrast (.learnmore[aria-label="Learn\ more"][target="_blank"])

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1531]
**User Experience:**
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Verify and check the luminosity ratio of 'Learn' more link.

**Actual Result:**
Luminosity ratio is less than 4.5:1 for 'learn more' link under 'Extensions' screen.

**Expected Result:**
Luminosity ratio is more than and equal to 4.5:1 for 'learn more' link under 'Extensions' screen.

**MAS Reference:**
[MAS 1.4.3 - Contrast (Minimum)] (https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Issue:**
 Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

**Target application title**: Microsoft MakeCode for micro:bit
Target application url: https://makecode.microbit.org/#editor

**Element path**: .card.link[role="link"]:nth-child(7) > .extra.hide.mobile > .learnmore[aria-label="Learn\ more"][target="_blank"]

**Snippet**: <a class="learnmore right floated" href="/pkg/tinkertanker/pxt-tinkercademy-tinker-kit" aria-label="Learn more" target="_blank" rel="noopener noreferrer">Learn more</a>

**How to fix:**
Fix any of the following:
  Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Environment:**
Chrome version 75.0.3770.142

This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights].

**Comment by sonimishra** [comment]
![MAS1 4 3_Extensions_CCA](https://user-images.githubusercontent.com/30456910/62525095-699a6300-b854-11e9-84d9-34190ed56c3f.jpg)

**Comment by sonimishra** [comment]
Bug is logged in incorrect 'monaco-editor' path. Hence closing this bug.
